recipes/CommonVoice/common_voice_prepare.py

```diff
@@ -236,20 +236,26 @@ def create_csv(
         # !! Language specific cleaning !!
         # Important: feel free to specify the text normalization
         # corresponding to your alphabet.
-
         if language in ["en", "fr", "it", "rw"]:
             words = re.sub(
                 "[^’'A-Za-z0-9À-ÖØ-öø-ÿЀ-ӿéæœâçèàûî]+", " ", words
             ).upper()
 
+        if language == "de":
+            words = words.replace("ß","0000ß0000")
```

Comment:
could you please add a comment explaining this line?

Reply:
Explanation added.